src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,125 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertFormatRequest = "INSERT INTO manufacturers (name, country) VALUES (?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createFormatStatement = connection
+                        .prepareStatement(insertFormatRequest, Statement.RETURN_GENERATED_KEYS)) {
+            createFormatStatement.setString(1, manufacturer.getName());
+            createFormatStatement.setString(2, manufacturer.getCountry());
+            createFormatStatement.executeUpdate();
+            ResultSet generatedKeys = createFormatStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not insert info to DB", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String getRequest = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                   PreparedStatement createFormatStatement =
+                        connection.prepareStatement(getRequest)) {
+            createFormatStatement.setLong(1, id);
+            ResultSet resultSet = createFormatStatement.executeQuery();
+            resultSet.next();
+            Manufacturer manufacturer = getManufacturer(resultSet);
```

Comment:
```suggestion
            if (resultSet.next()) {
                return Optional.of(getManufacturer(resultSet));
            }
```
It would be better to add check